core/crypto/vrf/p256/p256.go

```diff
@@ -123,14 +123,18 @@ func (k PrivateKey) Evaluate(m []byte) (vrf, proof []byte) {
 	// H = H1(m)
 	hx, hy := H1(m)
 
+	// VRF_k(m) = [k]H
+	vrfx, vrfy := params.ScalarMult(hx, hy, k.D.Bytes())
+
 	// G is the base point
-	// s = H2(m, [r]G, [r]H)
+	// s = H2(m, [r]G, [r]H, [k]H)
```

Comment:
Order is not following either the CONIKs paper or Franklin paper (which list [k]H earlier)

Also, why not hash in G (base point) and the public key kG? As is done in both papers